scripts/ingress_forward.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -ex 
+
+. scripts/include/common.sh
+. .envrc
+
+exec kubectl port-forward -n default pod/socksproxy 2224:8000
```

Comment:
:smiley:  Mind to add a default value for 2224 and make it overridable? Was about to do it but then I would collide with your changes

Reply:
done in https://github.com/SUSE/bkindwscf/commit/03e9467e0fcaff375e6da97cfb59332aa7e9840c, nevermind!